test/general/TestReachability.py

```diff
@@ -9,10 +9,33 @@
 class TestBase(unittest.TestCase):
     gen_steps = ["generate_early", "create_regions", "create_items", "set_rules", "generate_basic", "pre_fill"]
 
+    default_settings_unreachable_regions = {
+        "A Link to the Past": {
+            "Chris Houlihan Room",  # glitch room by definition
+            "Desert Northern Cliffs",  # on top of mountain, only reachable via OWG
+            "Dark Death Mountain Bunny Descent Area"  # OWG Mountain descent
+        },
+        "Ocarina of Time": {
+            "Prelude of Light Warp",  # Prelude is not progression by default
+            "Serenade of Water Warp",  # Serenade is not progression by default
+            "Lost Woods Mushroom Timeout",  # trade quest starts after this item
+            "ZD Eyeball Frog Timeout",  # trade quest starts after this item
+            "ZR Top of Waterfall",  # dummy region used for entrance shuffle
+        },
+        # The following SM regions are only used when the corresponding StartLocation option is selected (so not with default settings).
+        # Also, those dont have any entrances as they serve as starting Region (that's why they have to be excluded for testAllStateCanReachEverything).
+        "Super Metroid": {
+            "Ceres",
+            "Gauntlet Top",
+            "Mama Turtle"
+        }
+    }
+
     def testAllStateCanReachEverything(self):
         for game_name, world_type in AutoWorldRegister.world_types.items():
             # Final Fantasy logic is controlled by finalfantasyrandomizer.com
-            if game_name not in {"Ori and the Blind Forest"}:  # TODO: fix Ori Logic
+            if game_name != "Ori and the Blind Forest" and game_name != "Final Fantasy":  # TODO: fix Ori Logic
```

Comment:
This was reverted when merging main. Revert the reversion?
```suggestion
            if game_name not in {"Ori and the Blind Forest"}:  # TODO: fix Ori Logic
```